x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/activity_monitor/index.tsx

```diff
@@ -0,0 +1,337 @@
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 19,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 20,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+    {
+      id: 21,
+      rule: {
+        href: '#/detection-engine/rules/rule-details',
+        name: 'Automated exfiltration',
+      },
+      ran: '2019-12-28 00:00:00.000-05:00',
+      lookedBackTo: '2019-12-28 00:00:00.000-05:00',
+      status: 'Completed',
+      response: 'Success',
+    },
+  ];
+
+  const [itemsTotalState] = useState<number>(sampleTableData.length);
+  const [pageState, setPageState] = useState<PageTypes>({ index: 0, size: 20 });
+  // const [selectedState, setSelectedState] = useState<ColumnTypes[]>([]);
+  const [sortState, setSortState] = useState<SortTypes>({ field: 'ran', direction: 'desc' });
+
+  return (
+    <>
+      <EuiSpacer />
+
+      <EuiPanel>
+        <HeaderSection title="Activity monitor" />
+
+        <UtilityBar border>
+          <UtilityBarSection>
+            <UtilityBarGroup>
+              <UtilityBarText>{'Showing: 39 activites'}</UtilityBarText>
+            </UtilityBarGroup>
+
+            <UtilityBarGroup>
+              <UtilityBarText>{'Selected: 2 activities'}</UtilityBarText>
+
+              <UtilityBarAction iconType="stop">{'Stop selected'}</UtilityBarAction>
+            </UtilityBarGroup>
+
+            <UtilityBarGroup>
+              <UtilityBarAction iconType="cross">{'Clear 7 filters'}</UtilityBarAction>
+            </UtilityBarGroup>
+          </UtilityBarSection>
+        </UtilityBar>
+
+        <EuiBasicTable
+          columns={columns}
+          isSelectable
+          itemId="id"
+          items={sampleTableData}
+          onChange={({ page, sort }: { page: PageTypes; sort: SortTypes }) => {
+            setPageState(page);
+            setSortState(sort);
+          }}
+          pagination={{
+            pageIndex: pageState.index,
+            pageSize: pageState.size,
+            totalItemCount: itemsTotalState,
+            pageSizeOptions: [5, 10, 20],
+          }}
+          selection={{
+            selectable: (item: ColumnTypes) => item.status !== 'Completed',
+            selectableMessage: (selectable: boolean) =>
+              selectable ? undefined : 'Completed runs cannot be acted upon',
```

Comment:
Should your type be `selectable: boolean | undefined` above? You're checking for `undefined` below?

Reply:
As a side note, I almost always check for both `undefined` and `null` when I can and possible just to cover my bases with `selectable == null ? undefined : 'Completed runs cannot be acted upon'`

So many times I get burned by an unexpected null or other code changes. No changes required, just a general musing from my side.

Reply:
As above, everything under `activity_monitor/` is all the initial placeholder content added as part of https://github.com/elastic/kibana/pull/50220, so it'll be going away when it's time to implement that component.

And yeah, I agree `selectable != null` would be preferred here, or even `selectable ?? 'text'` with TS3.7.